README.md

```diff
@@ -1,10 +1,23 @@
 # Public Allocator
 
-```bash
-$ forge test
-```
-Running the tests this way will use this repository's compilation settings for dependencies (Morpho & MetaMorpho), potentially different from the settings used for deployment of those dependencies.
+## Overview
+
+[MetaMorpho](https://github.com/morpho-org/metamorpho) is a protocol for funds allocations and risk management on top of [Morpho Blue](https://github.com/morpho-org/morpho-blue). The Public Allocator is a contract that MetaMorpho curators can enabled as [allocator](https://github.com/morpho-org/metamorpho?tab=readme-ov-file#allocator) to let anybody reallocate the vault's funds to fill their liquidity needs.
+
+The Public Allocator's function `reallocateTo` ([source](https://github.com/morpho-org/public-allocator/blob/a43e2e29ba425a5e2af3dcb7c6774a4524abcda3/src/PublicAllocator.sol#L108), [interface](https://github.com/morpho-org/public-allocator/blob/cd82f57f2ca52c71fdb44ffb14278844beb593ec/src/interfaces/IPublicAllocator.sol#L62)) reallocates some liquidity from multiple markets to one market. It takes as input `withdrawals`, a list of `(MarketParams, uint128)` pairs, which will be the markets and amounts that will be reallocated, and `supplyMarketParams`, a `MarketParams`, which is the market in which to supply the reallocated funds. Note that the `id` of the markets in `withdrawals` must be sorted, and that `supplyMarketParams` cannot be a market of `withdrawals`.
```

Comment:
Not sure if we should go into the details of the interface. Better if we can avoid to say that `withdrawals` is a list of pairs, it can be confusing as `Withdrawal` is a struct